Framework/Kernel/inc/MantidKernel/V3D.h

```diff
@@ -2,6 +2,7 @@
 #define MANTID_KERNEL_V3D_H_
 
 #include <cmath>
+#include <cfloat>
```

Comment:
Why are you adding this to `V3D.h` when the undefined macro is in `Cylinder.cpp`?